### ISSUE TITLE
Trying to use boxes crashed game and corrupts world file

### ISSUE THREAD

**Issue by The-Konester** [BuiltBrokenModding/SBM-cardboardBoxes#10]
I have been using the mod for a while and have had no problems whatsoever until today when I updated my modlist, now whenever I try to use a box it fails to place it and the game crashes. on top of that, loading the chunk with the box in it crashes the world. the only conceivable issue I can imagine is a mod incompatibility, but I have no idea what mod it could be.

Here are multiple crash logs, I hope you can help me find the issue. lmk if I can provide anything else
[crash-2021-12-24_13.58.53-client.txt](https://github.com/BuiltBrokenModding/SBM-cardboardBoxes/files/7775322/crash-2021-12-24_13.58.53-client.txt)
[crash-2021-12-24_14.05.31-server.txt](https://github.com/BuiltBrokenModding/SBM-cardboardBoxes/files/7775323/crash-2021-12-24_14.05.31-server.txt)
[crash-2021-12-24_14.16.14-server.txt](https://github.com/BuiltBrokenModding/SBM-cardboardBoxes/files/7775324/crash-2021-12-24_14.16.14-server.txt)
[crash-2021-12-24_14.28.31-server.txt](https://github.com/BuiltBrokenModding/SBM-cardboardBoxes/files/7775325/crash-2021-12-24_14.28.31-server.txt)
[crash-2021-12-24_14.50.53-server.txt](https://github.com/BuiltBrokenModding/SBM-cardboardBoxes/files/7775326/crash-2021-12-24_14.50.53-server.txt)

**Comment by DarkGuardsman** [comment]
```
java.lang.NullPointerException: Exception ticking world
	at net.blay09.mods.cookingforblockheads.compat.CompatCapabilityLoader.attachTileEntityCapabilities(CompatCapabilityLoader.java:47) ~[?:9.3.4] {re:classloading}
```

Looks like an issue in compact machines but can't be sure

**Comment by dshadowwolf** [comment]
I've been seeing a non-fatal crash server side with that, but...

Placing a cardboard box on a spawner was not rendering that it was happening, even with spawners not blacklisted.

On going back to the location, the spawner was rendered with the cardboard box model, then the client crashed out.

Crash report for that: [crash-2022-02-15_14.57.44-client.txt](https://github.com/BuiltBrokenModding/SBM-cardboardBoxes/files/8074260/crash-2022-02-15_14.57.44-client.txt)

**Comment by dshadowwolf** [comment]
Looks like "Cooking for Blockheads" expects `TileEntity.getType()` to be non-null and for `TileEntity.getType().registryName()` to also be non-null

**Comment by bl4ckscor3** [comment]
Fixed in version 0.0.5. Sorry, this was caused by a stupid oversight on my part.
